tests/dopamine/replay_memory/circular_replay_buffer_test.py

```diff
@@ -704,13 +704,34 @@ def testAddTerminalNodeToTrajectoryBuffer(self):
         batch_size=BATCH_SIZE,
         contiguous_trajectories=True)
     self.assertEqual(memory.cursor(), 0)
-    self.assertEqual(len(memory._trajectories), 0)
+    self.assertEqual(len(memory._trajectory), 0)
     zeros = np.zeros(OBSERVATION_SHAPE)
     memory.add(zeros, 0, 0, 1)
     self.assertEqual(memory.cursor(), STACK_SIZE)
     self.assertEqual(memory.add_count, STACK_SIZE)
-    self.assertEqual(len(memory._trajectories), 0)
-    self.assertEqual(len(memory._trajectory_lengths), 0)
+    self.assertEqual(len(memory._trajectory), 0)
+
+  def testAddMultipleThreads(self):
```

Comment:
Per earlier comments, my own preference is to separate out the behaviors under test.  Should something break, it'll make the debugger's life a little easier.

Reply:
done. Split in two.